README.md

```diff
@@ -174,3 +175,23 @@ release:
   variables:
     INPUT_VERSION: yarn changeset version
 ```
+
+You may also want to run `yarn install` after the `changeset verion` command to update the yarn.lock in the version MR. You need to disable immutable lock file setting using an env variable.
```

Comment:
I'm not familiar with yarn p'n'p, does this mean the versions of current working packages are now stored in lock file at the same time? A bit wired to me. And what does `YARN_ENABLE_IMMUTABLE_INSTALLS` do and why it's required here?

Reply:
Yes yarn stores everything in the lock file including the workspace packages version.  
To prevent malicious code download and installs in the CI environments yarn prevents any change to the `yarn.lock` by default. (All CIs set an env called `CI=true` and yarn uses that)  
As `changesets-gitlab` is designed to run in CI environments and it changes the packages version when creating the MR and running the version command, this would be a common issue.

Reply:
> And what does YARN_ENABLE_IMMUTABLE_INSTALLS do and why it's required here?

@HosseinAgha

Reply:
`YARN_ENABLE_IMMUTABLE_INSTALLS=false` lifts this yarn restriction and let's us update the `yarn.lock` (in `yarn install` command) before creating the MR.